src/components/form/templates/array-field.ts

```diff
@@ -29,10 +29,10 @@ export class ArrayFieldTemplate extends React.Component {
         }
 
         return React.createElement('limel-button', {
-            outlined: true,
             label: 'Add',
             onClick: this.handleAddClick,
             icon: 'add',
+            class: 'button-add-new',
```

Comment:
Ah! Sorry, I didn't realise the information that this was an add-button existed in lime-elements already. I did see this file, but I missed that the icon and label properties shows the form knows this is an _Add_ button. Since it knows the context, it knows that a margin is needed 👍

Reply:
no worries 😄 so we're all good?

Reply:
Never anything but 😄 👍